README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
+   + here is the link to watch the best Trek while stuck inside. [You'll thank me](https://www.netflix.com/title/70158330?source=35) 
+* **Romantic comedies**
+    + Before you ask, it's *You've Got Mail* and I do not like *Sleepless in Seattle*
+---------
+I love travelling--before coming to the MAPSS program, I travelled for four months--one of my favorite places was Georgia--but I didn't take any photos of me there so here's one from Armenia.   
+![picture of me at the top of Noravank Monastery in Armenia](https://avatars0.githubusercontent.com/u/63277774?s=400&u=48a639302371ba85f861f97c50cd96cdd10e7844&v=4)  
+----
+Right now, my GitHub workflow is probably a little clunky. Since i am working on specific, small skills, I am constantly pushing it to Git to see if my code did what I wanted it to do. I have so many pushes for this assignment! I'm a little worried moving forward about not being to see if what I have written works as I go, but I'm sure I'll get used to it.  
```

Comment:
Just note if you want paragraph breaks, you need to leave a blank line between paragraphs.

```
Right now, my GitHub workflow is probably a little clunky. Since i am working on specific, small skills, I am constantly pushing it to Git to see if my code did what I wanted it to do. I have so many pushes for this assignment! I'm a little worried moving forward about not being to see if what I have written works as I go, but I'm sure I'll get used to it.  
RMarkdown right now seems a tad overwhelming with all of the commands--I just need to remember that it is a different language! So far, these kinds of commands seem simple enough though.
```

Right now, my GitHub workflow is probably a little clunky. Since i am working on specific, small skills, I am constantly pushing it to Git to see if my code did what I wanted it to do. I have so many pushes for this assignment! I'm a little worried moving forward about not being to see if what I have written works as I go, but I'm sure I'll get used to it.  
RMarkdown right now seems a tad overwhelming with all of the commands--I just need to remember that it is a different language! So far, these kinds of commands seem simple enough though.

vs.

```
Right now, my GitHub workflow is probably a little clunky. Since i am working on specific, small skills, I am constantly pushing it to Git to see if my code did what I wanted it to do. I have so many pushes for this assignment! I'm a little worried moving forward about not being to see if what I have written works as I go, but I'm sure I'll get used to it.  

RMarkdown right now seems a tad overwhelming with all of the commands--I just need to remember that it is a different language! So far, these kinds of commands seem simple enough though.
```

Right now, my GitHub workflow is probably a little clunky. Since i am working on specific, small skills, I am constantly pushing it to Git to see if my code did what I wanted it to do. I have so many pushes for this assignment! I'm a little worried moving forward about not being to see if what I have written works as I go, but I'm sure I'll get used to it.  

RMarkdown right now seems a tad overwhelming with all of the commands--I just need to remember that it is a different language! So far, these kinds of commands seem simple enough though.